src/Generators/Microsoft.Gen.AutoClient/Roslyn4.0/Microsoft.Gen.AutoClient.Roslyn4.0.csproj

```diff
@@ -20,10 +20,4 @@
       <DependentUpon>Resources.resx</DependentUpon>
     </Compile>
   </ItemGroup>
-
-  <ItemGroup>
-    <PackageVersion Update="Microsoft.CodeAnalysis.CSharp.Workspaces" Version="4.0.1" />
```

Comment:
All of these generators are specifically trying to build against 4.0, so we shouldn't bump them to 4.4 as that makes them not be supported any longer on older toolset versions.